### PR TITLE
Filters moved to the store. Dynamic ListPage construction from the config json-file

### PR DESCRIPTION
**Pull request by OlegMakarenko**
- **Dropdown** options and current value moved to the store
- Added **PageList** component which will be used as a template
- Added config file `src/config/list-pages.json` for the router and PageList layout instructions

Resolved: #246

### CODE REVIEW

File: src/config/list-pages.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+    "pages": [
+        {
+            "path": "/blocks",
+            "name": "blocks",
+            "props": {
+              "namespace": "block",
```
Comment (AnthonyLaw) [inline src/config/list-pages.json]: Just my opinion, show change the name "namespace" to "storeNamespace".
incase miss understand, `mosaic namespace`

File: src/router.js [hunk 2]
```diff
@@ -18,78 +18,87 @@
...
       component: () =>
         import('./views/Home.vue')
     },
-    {
-      path: '/blocks',
-      name: 'blocks',
-      component: () =>
-        import('./views/Blocks.vue')
-    },
+    // {
```
Comment (AnthonyLaw) [inline src/router.js]: i think just remove the unused code.

File: src/store/transaction.js [hunk 3]
```diff
@@ -79,15 +79,23 @@ export default {
...
+      height: el.blockHeight,
+      deadline: el.deadline,
+      transactionHash: el.transactionHash,
+      type: el.transactionBody.type,
+      fee: el.fee,
+      signer: el.signer,
+      recipient: el.transactionBody.recipient
     })),
+    // getTimelineFormatted: (state, getters) => getters.getTimeline.current.map(el => ({
+    //   height: el.height,
```
Comment (AnthonyLaw) [inline src/store/transaction.js]: i think should remove as well, if it's not longer using.

File: src/config/list-pages.json (hunk 1 above)
Comment (OlegMakarenko) [reply src/config/list-pages.json]: Good suggestion, agree

**Pull request merged**